src/lib/experimental.ts

```diff
@@ -12,12 +12,16 @@ export interface IExperimentalSegments {
     enableSegmentsClientApi: boolean;
     enableSegmentsAdminApi: boolean;
     inlineSegmentConstraints: boolean;
+    segmentValuesLimit: number;
+    strategySegmentsLimit: number;
 }
 
 export const experimentalSegmentsConfig = (): IExperimentalSegments => {
     return {
         enableSegmentsAdminApi: true,
         enableSegmentsClientApi: true,
         inlineSegmentConstraints: true,
+        segmentValuesLimit: 100,
```

Comment:
My gut feel is that these numbers might be better to shift out either to a local constant or into the constants file. Probably not a big deal since it's in a single place